contracts/ValidatorSet.template

```diff
@@ -18,7 +18,7 @@ contract ValidatorSet is IValidatorSet, System, IParamSubscriber {
   using RLPDecode for RLPDecode.Iterator;
   using RLPDecode for RLPDecode.RLPItem;
 
-  uint256 public constant BLOCK_REWARD = 3e18;
+  uint256 public constant BLOCK_REWARD = 3e18; //@openissue
```

Comment:
'3 ether' is more readable

Reply:
Opinion: I prefer 3e18 - while we're EVM compatible, I feel like this change would be better if the contracts were deployed to Ethereum as opposed to Core.